### ISSUE TITLE
Host application skipping Collective selection step

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#4278]
If you go to https://opencollective.com/opensource and click `apply` you see this modal which lets you select an existing Collective or create a new one.

<img width="1214" alt="Screen Shot 2021-05-13 at 9 51 12 AM" src="https://user-images.githubusercontent.com/5498878/118048733-cf743100-b3d0-11eb-85fd-9515f7e69ee1.png">

But if you go directly to https://opencollective.com/opensource/apply you see this:

<img width="599" alt="Screen Shot 2021-05-13 at 9 52 29 AM" src="https://user-images.githubusercontent.com/5498878/118048962-309c0480-b3d1-11eb-9ede-eec4eaa2aa4b.png">

Selecting `manual verification` takes you here:

<img width="603" alt="Screen Shot 2021-05-13 at 9 53 05 AM" src="https://user-images.githubusercontent.com/5498878/118049002-3db8f380-b3d1-11eb-9fa2-d23fab25be83.png">

So your only option is to create a new Collective. The https://opencollective.com/opensource/apply path should include the modal where you can select an existing Collective.

**Comment by znarf** [comment]
Why would they go the apply page directly?

This would change completely the design of a flow that is existing for many years.

Not possible for engineering to tackle this on our own.

**Comment by alanna** [comment]
Because we often directly link people to https://opencollective.com/opensource/apply

Sorry I don't understand the technical implications @znarf , but why can't the above URL just take you to the screen that pops up if you click the apply button, where you can select the Collective, and then to the screen asking to select manual or github verification and go on from there as the flow is now? Is it because the github verification flow always includes creating a new Collective, not applying with an existing one?

**Comment by znarf** [comment]
@alanna Sorry, I might have misunderstood the proposal and thought it was more complicated. Let me rephrase what I understand now:

1. Change the URL of the "Apply with a new Collective" flow and move it from https://opencollective.com/opensource/apply to https://opencollective.com/opensource/apply/new That's necessary for step 2).
2. Now that https://opencollective.com/opensource/apply is free, change its behavior and simply display the Collective Page  with the "Apply" modal opened.

Looks doable without Product or Design involvement.

The GitHub flow would not be impacted, but you're right, it's only for new Collectives. Existing Collectives need to apply manually. We should not try to change this at this point, or at least not in this issue.

**Comment by alanna** [comment]
Yes that sounds like a good solution.

**Comment by znarf** [comment]
Spec (copy pasted from [past comment](https://github.com/opencollective/opencollective/issues/4278#issuecomment-842273338)):

1. Change the URL of the "Apply with a new Collective" flow and move it from https://opencollective.com/opensource/apply to https://opencollective.com/opensource/apply/new That's necessary for step 2).
2. Now that https://opencollective.com/opensource/apply is free, change its behavior and simply display the Collective Page with the "Apply" modal opened.

**Comment by znarf** [comment]
A $200 bounty was attached to this issue. Anyone submitting a Pull Request will be rewarded with $200 when the Pull Request is reviewed, accepted and merged. [More info](https://docs.opencollective.com/help/contributing/development/bounties).

This is issue is marked as "medium complexity" and is reserved for people with enough experience with the code base, at least one fully completed issue.

**Comment by znarf** [comment]
🏆 This issue was completed by @snitin315 

To claim the bounty please, as a recurring contributor, please simply add it to your next invoice to Open Collective Engineering https://opencollective.com/engineering/expenses/new  Please make sure to mention the ticket number in the invoice and in the expense title.
